src/getHumanAge.test.js

```diff
@@ -3,6 +3,90 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
+  test(' if value catAge and dogAge is 14 and 14 output [0, 0]', () => {
+    const humanAge = getHumanAge(14, 14);
+
+    expect(humanAge)
+      .toEqual([0, 0]);
+  });
+
+  test(' if value catAge and dogAge is 15 and 15 output [1, 1]', () => {
```

Comment:
```suggestion
  test(' should return [1, 1] when cat and dog age is equal to 15', () => {
```
we should write test name in such way